lib/abr/simple_abr_manager.js

```diff
@@ -243,6 +264,30 @@ shaka.abr.SimpleAbrManager = class {
   }
 
 
+  /**
+   * @override
+   * @export
+   */
+  setMediaElement(mediaElement) {
+    this.mediaElement_ = mediaElement;
+    if (this.resizeObserver_) {
+      this.resizeObserver_.disconnect();
+      this.resizeObserver_ = null;
+    }
+    if (this.mediaElement_ && 'ResizeObserver' in window) {
```

Comment:
Should there be documentation calling out the fact that this feature will not work on browsers that don't support `ResizeObserver`?

Reply:
If you look at externs it is documented. https://github.com/shaka-project/shaka-player/pull/4404/files#diff-9ac7cefa529d82513af9ff09186f9a6de19b063416e27c6fe97e834ea223388aR1036